File: sprint4/By languages/Java/lesson1code1.java
// Словарь "страна по городу"
Map<String, String> countryByCity = Map.of(
    "Moscow", "Russia",
    "Tula", "Russia",
    "London", "UK",
    "Paris", "France",
    "Leon", "France",
    "San Francisco", "USA",
    "New York", "USA"
);

// Присвоить значение по ключу
countryByCity.put("Saint Petersburg", "Russia");
// Получить значение по ключу
String country = countryByCity.get("Tula");
System.out.println("Tula is located in " + country);

File: sprint4/By languages/python/lesson1code1.py
country_by_city = {
    "Moscow": "Russia",
    "Tula": "Russia",
    "London": "UK",
    "Paris": "France",
    "Leon": "France",
    "San Francisco": "USA",
    "New York": "USA",
}

# присвоить значение по ключу
country_by_city["Saint Petersburg"] = "Russia"
# получить значение по ключу
country = country_by_city["Tula"]
print(f"Tula is located in {country}")